Formula/fzf.rb

```diff
@@ -5,6 +5,7 @@ class Fzf < Formula
   sha256 "ce76f5f1a7cef05061e3cb2ab1eba7bc47660a868d622b5dd914e50158129ff6"
   license "MIT"
   head "https://github.com/junegunn/fzf.git"
+  revision 1
```

Comment:
That will not pass `brew audit --strict fzf`. `revision` line should come before `head`. Please follow the instructions in https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#to-contribute-a-fix-to-the-foo-formula.